pkgs/by-name/fl/flottbot/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nix-update-script
+, substituteAll
+}:
+buildGoModule rec {
+  pname = "flottbot";
+  version = "0.13.0";
+
+  src = fetchFromGitHub {
+    owner = "target";
+    repo = pname;
```

Comment:
usage of pname is considered an anti pattern https://github.com/NixOS/nixpkgs/issues/277994

Reply:
Didn't know about this, thanks!